DuckDuckGo/TabManager.swift

```diff
@@ -25,7 +25,7 @@ class TabManager {
 
     private(set) var model: TabsModel
     
-    private var tabControllerCache = [TabViewController]()
+    private(set) var tabControllerCache = [TabViewController]()
```

Comment:
I originally added a new function which was basically a custom `forEach` on `TabManager` which ran a given block over each `TabViewController`, but that wasn't much different to making this readable publicly. Open to changes here though. 🙂